cmd/resource_cmds.go

```diff
@@ -0,0 +1,39 @@
+// this file WILL be generated (sc-241153)
+
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"ldcli/cmd/resources"
+)
+
+func addAllResourceCmds(rootCmd *cobra.Command, client *http.Client) {
+	// Resource commands
+	gen_TeamsResourceCmd := resources.NewResourceCmd(
+		rootCmd,
+		"teams",
+		"A team is a group of members in your LaunchDarkly account.",
+		"A team can have maintainers who are able to add and remove team members. It also can have custom roles assigned to it that allows shared access to those roles for all team members. To learn more, read [Teams](https://docs.launchdarkly.com/home/teams).\n\nThe Teams API allows you to create, read, update, and delete a team.\n\nSeveral of the endpoints in the Teams API require one or more member IDs. The member ID is returned as part of the [List account members](/tag/Account-members#operation/getMembers) response. It is the `_id` field of each element in the `items` array.",
```

Comment:
this is just a snippet from the teams tag description, which comes from [here](https://github.com/launchdarkly/gonfalon/blob/main/apidocs/content/teams.md) 

I stripped out the header manually, hoping that the format holds true for the other resources (looking like mayybe it is? Will check with Molly).